src/app.ts

```diff
@@ -37,4 +37,8 @@ app.get('/', (req: express.Request, res: express.Response) =>  {
   res.render('index.njk');
 });
 
-app.get('/test', (() => getAllDatabases));
\ No newline at end of file
+app.get('/test', (req, res, next) => {
+  getAllDatabases(req, res).catch((err:unknown) => {
+    next(err);
+  });
+})
```

Comment:
Teachable Moment time:

Sometimes, linters make us do things we disagree on. I prefer to put it back the way Express wants it and have Lint ignore a specific rule for the file.

It's not an instruction to disable lint everywhere, but feel free to disable it for these.

Reply:
This has been commented and corrected.